### PR TITLE
A very simple sample for using 3DCursorLibrary

### PR DESCRIPTION
**Pull request by parthmehta209**
depends on https://github.com/Samsung/GearVRf/pull/628

GearVRf-DCO-1.0-Signed-off-by: Parth Mehta <parth.mehta@samsung.com>

### CODE REVIEW

**Comment by parthmehta209** [comment]
@rahul27 review please.

File: gvr-3dcursor-simple/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    lintOptions {
+        checkReleaseBuilds false
+    }
+}
+
+dependencies {
+    compile fileTree(include: ['*.jar'], dir: 'libs')
+    compile project(':3DCursorLibrary')
+    //_VENDOR_TODO_ add the device to cube-app here
```
Comment (rahul27) [inline gvr-3dcursor-simple/app/build.gradle]: remove two lines.

File: gvr-3dcursor-simple/settings.gradle [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+include 'app'
+include ':framework'
+include ':3DCursorLibrary'
+
+//_VENDOR_TODO_ add your device folder here for Android Studio to locate it
```
Comment (rahul27) [inline gvr-3dcursor-simple/settings.gradle]: remove lines

File: gvr-3dcursor-simple/app/build.gradle (hunk 1 above)
Comment (parthmehta209) [inline gvr-3dcursor-simple/app/build.gradle]: Done

File: gvr-3dcursor-simple/settings.gradle (hunk 2 above)
Comment (parthmehta209) [inline gvr-3dcursor-simple/settings.gradle]: Done

**Comment by NolaDonato** [comment]
Does this replace the 3dcursor sample? If not then there is a conflict in Eclipse. It won't let you import both 3dcursor and 3dcursor-simple because they have the same project name - CursorActivity

**Comment by parthmehta209** [comment]
This does not replace the 3dcursor sample. I have not tested in eclipse. Will fix the name.

**Comment by parthmehta209** [comment]
Not possible to build it using eclipse as it uses the 3DCursor extension which does not support an eclipse build.

**Comment by liaxim** [comment]
@rahul27 @NolaDonato Can we merge this?

**Comment by NolaDonato** [comment]
Yes, I tested it. Looks good.

**Pull request merged**